File: CPP/src/DogMeat.cpp
#include "../include/DogMeat.hpp"

//Constructor menerima harga, dan mempassing producer untuk superclass.
DogMeat::DogMeat(int _price, string _producer){
	initializeFarmProduct(_producer);
	initializeProduct(_price, FARMPRODUCT, DOG_MEAT, "Daging Anjing");
}

File: Java/src/main/java/engisubs/gameobject/product/farm_product/meat/DogMeat.java
package main.java.engisubs.gameobject.product.farm_product.meat;

public class DogMeat extends Meat{
    public DogMeat(int _price, String _producer){
        super.initializeFarmProduct(_producer);
        super.initializeProduct(_price, ProductType.FARMPRODUCT, GameObjectType.DOG_MEAT, "Daging Segar Anjing");
    }
}